types.go

```diff
@@ -62,6 +62,17 @@ const (
 	OrderIntentAuthorize string = "AUTHORIZE"
 )
 
+// Possible value for `status` in GetOrder
+//
+// https://developer.paypal.com/docs/api/orders/v2/#orders-get-response
+const (
+	OrderStatusCreated   string = "CREATED"
```

Comment:
Is `string` really necessary here?

Reply:
No, however the rest of the file is formatted like that, so I included it for consistency.